examples/do-not-use/main.tf

```diff
@@ -2097,6 +2092,29 @@ resource "aws_cloudwatch_log_group" "fargate_fluentbit" {
   tags              = merge(var.tags, try(var.fargate_fluentbit_cw_log_group.tags, {}))
 }
 
+resource "aws_iam_policy" "fargate_fluentbit" {
```

Comment:
where is this policy used? we need to build out a better policy - as it currently stands, if you deploy two clusters in the same account, this will fail with a name collision

Reply:
Without this policy, logGroups are not populated with logStreams. 
I'll fix the name to match the cluster.

Reply:
That will introduce a number of issues due to the length of cluster names that customers use
- https://github.com/aws-ia/terraform-aws-eks-blueprints/issues/333
- https://github.com/aws-ia/terraform-aws-eks-blueprints/issues/1183